tsconfig.json

```diff
@@ -11,5 +11,5 @@
   "include": [
     "media.d.ts",
     "src/**/*"
-  ]
+, "src/components/resume"  ]
```

Comment:
why have you added a single file? this is handeld by the `src/**/*` import

Reply:
I didn't recall adding this, but it has been removed after your note